Frameworks/include/StringHelpers.h

```diff
@@ -465,9 +466,22 @@ inline TSource ContainerFromDelimitedString(_In_ const std::wstring& delimitedSt
     return result;
 }
 
-// Returns a a vector from a delmited string
+// Returns a vector from a delmited string
 inline std::vector<std::wstring> VectorFromDelimitedString(_In_ const std::wstring& delimitedString, wchar_t delimiter) {
     return ContainerFromDelimitedString<std::vector<std::wstring>>(delimitedString, delimiter);
 }
 
+#ifdef __OBJC__
```

Comment:
I want you to find the error and think about it more. I have a sneaking suspicion that the error is ore related to our #defs etc for clang than for objective c as a language. I bet if you built for c++ in clang it would work fine. This mostly points to a failing in our sbclang.props / .targets in that its not symmetric building clang and cl but i'd like us to now overindex on this difference as it makes fixing it even harder as each of these things needs to be re-evaluated for what they really are trying to do.  

perhaps an ifdef for clang? or something else related to having CoreFoundation includes available?

---
In reply to: [90938607](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90938607) [](ancestors = 90938607,90742443)